README.md

```diff
@@ -191,11 +191,11 @@ specify that by adding an `aws_partition` option as below.
 ```ini
```

Comment:
Just one final small thing avoid confusion. Might wanna change the aws_partition to aws-partition at line 189.